build.gradle

```diff
@@ -70,7 +70,6 @@ subprojects {
       testImplementation "org.hamcrest:hamcrest-core"
       testRuntimeOnly "cglib:cglib-nodep"
       testRuntimeOnly "org.objenesis:objenesis"
-      testImplementation "org.codehaus.groovy:groovy-all"
```

Comment:
I verified that the same tests run both before and after this change.  There are some tests not running on master already which we'll address in a followup PR.